protos/fpc.options

```diff
@@ -1,8 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
-fpc.CleartextChaincodeRequest.return_encryption_key type:FT_POINTER
-
 fpc.ChaincodeRequestMessage.encrypted_request type:FT_POINTER
+fpc.ChaincodeRequestMessage.encrypted_key_transport_message type:FT_POINTER
+
+fpc.KeyTransportMessage.request_encryption_key type:FT_POINTER
+fpc.KeyTransportMessage.response_encryption_key type:FT_POINTER
```

Comment:
These should in principle be fixed size?

Reply:
True, but keeping this variable allows to decide the crypto lib the size of the keys.